portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,16 +19,31 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-    
+
+    private ArrayList<String> outputStrings = new ArrayList<String>();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Sara!");
+   
+    outputStrings.add("I love salamanders");
+    outputStrings.add("Macarons are my favorite food");
+    outputStrings.add("How do you like these apples");
```

Comment:
I would just move this to the initialization of the array.
private final List<String> outputStrings = new ArrayList<>("I love...", "Macatons...", "How do you...")

Reply:
Done

Reply:
This syntax is giving me syntax errors, no type conversion from List to ArrayList and it's not allowing me to init values in the ()